ci/run_tests_test.go

```diff
@@ -12,17 +12,25 @@ import (
 )
 
 const (
-	// qf101 is a test course for experimenting with go.sh behavior.
-	// The test below will run locally on the test machine, not on the QuickFeed machine.
-	getURL  = "https://github.com/qf101/meling-labs.git"
-	testURL = "https://github.com/qf101/tests.git"
+	gh         = "github.com"
+	qf101      = "qf101"
+	ghUserName = "meling"
```

Comment:
Fetch `GITHUB_USER` from the environment, and fallback to `USER` if not defined.